### DISCUSSION TITLE
Suggestion: pls make it possible to read prompt for ai image

### DISCUSSION THREAD

**Discussion by lostidogi** [d2phap/ImageGlass#1578] [General] [answered]
this will be cool if you add some prompt reader in ImageGlass (like png info on stable diffusion)

thanks

**Comment by d2phap** [comment, marked as answer]
You can use Exiftool (v8.8) or [ExifGlass](https://github.com/d2phap/ExifGlass) (v9.0 beta 4) to read the metadata from image with the custom format:
See https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/1569#discussioncomment-4175458

**Reply by lostidogi** [reply]
ohh okay cool thank you :)
